test/packages/parallel/httpcheck.stack_version

```diff
@@ -0,0 +1 @@
+9.2.0-SNAPSHOT
```

Comment:
Forced to run this package with 9.2.0-SNAPSHOT.

This should be removed once the default version is 9.2.0 or higher.